en/part2.md

```diff
@@ -22,21 +22,21 @@ Simply put, TCP can be utilized better to avoid pauses or wasted intervals that
 
 ## 2.4 Transfer sizes and number of objects
 
-When looking at the trend for some of the most popular sites on the web today and what it takes to download their front pages, a clear pattern emerges. Over the years, the amount of data that needs to be retrieved has gradually risen up to and above 1.9MB. What is more important in this context is that, on average, over 100 individual resources are required to display each page.
+When looking at the trend for some of the most popular sites on the web today and what it takes to download their front pages, a clear pattern emerges. Over the years, the amount of data that needs to be retrieved has gradually risen up to and above 1.9 MB. What is more important in this context is that, on average, over 100 individual resources are required to display each page.
```

Comment:
Here is specified "_1.9 MB_", but on `transfer size growth` graph indicated 2.1 MB.
It's ok?